DatadogFlags/Sources/Flags.swift

```diff
@@ -28,14 +28,32 @@ public enum Flags {
         /// Default: `nil`.
         public var customExposureEndpoint: URL?
 
+        /// Enables exposure logging via the dedicated exposures intake endpoint.
+        ///
+        /// When enabled, flag evaluation events are sent to the exposures endpoint for analytics and monitoring.
+        ///
+        /// Default: `true`.
+        public var enableExposureLogging: Bool
+
+        /// Enables exposure logging via RUM integration.
+        ///
+        /// When enabled, flag evaluation events are logged as RUM actions for correlation with user sessions.
+        ///
+        /// Default: `true`.
+        public var enableRUMIntegration: Bool
```

Comment:
Suggestion: Consider using `trackExposures` and `rumIntegrationEnabled` instead of `enableExposureLogging` and `enableRUMIntegration` to align with the SDK's configuration naming conventions. RUM uses `track*` prefix for collecting events (`trackFrustrations`, `trackBackgroundEvents`, etc.) and `*Enabled` suffix for integrations (`bundleWithRumEnabled`, `networkInfoEnabled`). This would make the naming more consistent across the codebase.

Reply:
Thanks! I hadn't noticed this pattern in naming conventions, but I agree that we should aim to be consistent

https://github.com/DataDog/dd-sdk-ios/pull/2510/commits/30342b0606c6db2247f4f5e0f6844629d357c9f1